zerver/webhooks/gitlab/view.py

```diff
@@ -318,19 +318,17 @@ def get_subject_based_on_event(event: str, payload: Dict[str, Any]) -> Text:
     return get_repo_name(payload)
 
 def get_event(request: HttpRequest, payload: Dict[str, Any], branches: Optional[Text]) -> Optional[str]:
+    # if there is no 'action' attribute, then this is a test payload
+    # and we should ignore it
     event = request.META['HTTP_X_GITLAB_EVENT']
-    if event == 'Issue Hook':
+    if event in ['Issue Hook', 'Merge Request Hook', 'Wiki Page Hook']:
         action = payload['object_attributes'].get('action')
+        if action is None:
+            return None
```

Comment:
We should probably send a "webhook configuration success" message for this payload.